embedding-calculator/src/services/facescan/plugins/facenet/facenet.py

```diff
@@ -105,11 +99,43 @@ def find_faces(self, img: Array3D, det_prob_threshold: float = None) -> List[Bou
             filtered_bounding_boxes.append(box)
         return filtered_bounding_boxes
 
+
+class Calculator(base.BaseCalculator):
+    ml_models = (
+        # links from https://github.com/davidsandberg/facenet#pre-trained-models
+        # VGGFace2 training set, 0.9965 LFW accuracy
+        ('20180402-114759', 'https://drive.google.com/uc?id=1EXPBSXwTaqrSC0OhUdXNmKSh9qJUQ55-'),
```

Comment:
Are this models stored on our google drive? If not - let's put them to our google drive